docs/Versioning.md

```diff
@@ -1,142 +1,152 @@
+
+## Model versioning
+
+Model versioning is ultimately the domain of a given organization. Therefore, this section of the specification is not normative. It simply outlines a set of recommended practices.
+
+Model authors and applications/systems MAY elect to ignore the model versioning mechanism and policy rules. For models that will be shared across developers, teams, or organizations, model authors and applications/systems SHOULD adhere to the following version policies:
+
+ISSUE: the following is a strawman. I'm confident some of it is right and some is wrong. Either way, we need to make some calls and document it.  Also note that A LOT of this will likely apply to operators, as both operators and graphs have signatures AND can be versioned.
+
+### Signature Changes
+
+1. Breaking changes to the ModelProto.graph.GraphProto.input or .output MUST increment the MAJOR version of `ModelProto.model_version`. Breaking changes include:
+
+    * Breaking changes to the semantics of an input or output (e.g., changing the required contents of an input tensor from a color image to a black and white image).
+    * Changing the declared type of an input or output to an incompatible type (e.g., `tensor(int)->tensor(string)`).
+    * Adding a new input for which there is no meaningful or specified default value. For graph inputs, those values are provided by a same-named value in GraphProto. initializer.
+    * Removing an existing output for which there is no meaningful or specified default value.
+
+2. Non-breaking changes to the ModelProto.graph.GraphProto.input or .output MUST increment the MINOR version of `ModelProto.model_version`. Non-breaking changes include:
+
+    * Changing the declared type of an input or output to an compatible/widening type (e.g., `tensor(int32)->tensor(int64)`, `tensor(float16)->tensor(float32)`.
+    * Adding a new input for which there is a meaningful or specified default value.
+    * Adding new behavior that is only triggered in the presence of inputs that were not
+    possible in prior versions of the graph (typically by the presence of a new input
+    or allowing a previously invalid input value).
+
+### IR version/Operator version dependency changes
+
+ISSUE: what's our policy when a model takes a dependency on new `IR_VERSION` change and/or new operator change?
+
+
+### Accuracy or performance changes
+
+Assuming that there are no breaking changes to the signature of the model's graph or any operator dependencies, the shape and contents of the graph can change freely provided there are no semantic changes to the model. However, changes to the shape and contents of the graph can impact model accuracy and/or model performance.
+
+## Released Versions
+
+ONNX version|File format version|Operator set version ai.onnx|Operator set version ai.onnx.ml

```

Comment:
Can we keep this IR Version unless we update it everywhere?

Reply:
@smk2007 : What do you mean? This PR is just documenting the relationship to existing versions, not changing the file format version.